### DISCUSSION TITLE
Problem with repetitions

### DISCUSSION THREAD

**Discussion by lukasc-ubc** [openai/whisper#2442] [General]
I am transcribing audio to text using Whisper and the Aiko app.  About 30% of the way through a 1 hour recording, it gets stuck in a loop, and outputs: 
Thank you. Thank you.  Thank you.

I tried a setting in the app, "Reduce repetitions", and it get a different repetitive pattern, again stuck in a loop.

The above was on Mac OSX M1 processor, using v2 of the model.

I also tried with the iPhone.  and it got stuck on "Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, "

**Comment by glangford** [comment]
Unfortunately this is a well known issue that hasn't been resolved. See this past discussion for an explanation and possible workarounds.

* https://github.com/openai/whisper/discussions/192

**Comment by lukasc-ubc** [comment]
@glangford 

Thank you for your reply.  That discussion topic was 2 years ago, and doesn't seem to have a resolution.  I had a colleague run my audio file using the Python API, and it also had problems, but generated some different repetition text.  Shouldn't this have been fixed in the model?  

I imagine the Aiko app uses that suggestion in their option for reducing repetitions, but I'll ask.

**Reply by glangford** [reply]
> Shouldn't this have been fixed in the model?

There are new models you can experiment with (large-v3, turbo) but the underlying problem has not been fixed as far as I am aware.

**Comment by Purfview** [comment]
Check this bugfix: https://github.com/openai/whisper/pull/1903

**Comment by realtechspecs** [comment]
> I am transcribing audio to text using Whisper and the Aiko app. About 30% of the way through a 1 hour recording, it gets stuck in a loop, and outputs: Thank you. Thank you. Thank you.
> 
> I tried a setting in the app, "Reduce repetitions", and it get a different repetitive pattern, again stuck in a loop.
> 
> The above was on Mac OSX M1 processor, using v2 of the model.
> 
> I also tried with the iPhone. and it got stuck on "Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, Uh, "

Can you upload the audio/video file you used

**Comment by joostiphone** [comment]
'Skip Silent Parts' is a relatively new setting in Aiko which is a setting in beta phase. This worked for me.
